docs/design/proposal-troubleshoot-consolidated-cli.md

```diff
@@ -0,0 +1,166 @@
+# Consolidated Troubleshoot CLI
+
+## Goals
+
+- Consolidate all top level Troubleshoot commands (`support-bundle`, `preflight`, `analyze`, `redact`, and `sbctl`) into one exposing subcommands of all the functionality that was implemented by the previous commands.
+- Ensure functional backward compatibility of newly introduces interfaces (CLI and Public APIs)
+
+## Non-Goals
+
+- Maintain backward compatibility of the CLI interface and the public APIs (new interfaces will be created)
```

Comment:
let's add a note to the effect that the existing CLI will remain supported until such time as the new CLI is in common use at least by the Replicated product set.